redisdb/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG - redisdb
 
+1.1.1 / 2017-08-11
```

Comment:
The change has not been released yet so use `Unreleased`. Plus it's not a bug fix but a new feature, so the right version is `1.2.0`.

```
1.2.0 / Unreleased
```